### DISCUSSION TITLE
Error with Enabling Supersaturation for oral absorption model of a BCS class II drug

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1741] [Questions & Problems]
Hello OSP Community,

Currently, I'm working on a PBPK model for oral absorption of an IR tablet of a drug similar to Citazole, which falls under BCS Class II, having poor solubility and high permeability. I have come across a technical problem related to the "Supersaturation" parameter in the advanced settings within the compound tab, as follows: 

Initially, I enabled the "Supersaturation" option for an existing simulation, which worked as expected. However, when I tried to create a new simulation or clone an existing one with supersaturation enabled, I got the following error message:

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/67606842/93a2c54c-6028-4d35-87f9-ae1d80d19439)

We assumed to enable supersaturation based on the methodology described in a publication for modelling poorly soluble compounds (https://pubmed.ncbi.nlm.nih.gov/20554023/), which suggests that supersaturation can be a critical factor for such class of drugs.

Could someone please explain why this error occurs when creating or cloning simulations with supersaturation enabled? Am I missing a specific step or consideration here? 
Any guidance on how to correctly implement supersaturation for BCS Class II drugs in PK-Sim would be greatly appreciated.

Thank you in advance, 
Aya

**Comment by Yuri05** [comment]
do you have oral formulation other than "Particles dissolution" in your simulation?

**Reply by AyaSaleh90** [reply]
Actually the only formulation we have is film-coated tablet, and its dissolution characteristics have been parameterised using the Weibull function. We choose that function based on previous PBPK models for the same compound and the absence of the in vitro dissolution data for this formulation.
I have read in the documentation that Supersaturation only works with "particle dissolution", but what would be the case if we dont have in vitro information data required as input for such formulation?

**Reply by AndreDlm** [reply]
Supersaturation itself is not critical, it's rather the subsequent precipitation that can occur (and that can only be modeled with the particle dissolution model). However, if you want to use supersaturation without subsequent precipitation, you could just increase the (intestinal) solubility of your drug to get the same effect
